geonode/base/models.py

```diff
@@ -545,8 +517,6 @@ class ThesaurusKeyword(models.Model):
 
     thesaurus = models.ForeignKey("Thesaurus", related_name="thesaurus", on_delete=models.CASCADE)
 
-    image = models.CharField(max_length=512, help_text="A URL to an image", null=True, blank=True)
```

Comment:
@mwallschlaeger why did you drop this field? It was introduced in https://github.com/GeoNode/geonode/commit/c437a0a69054217976fa519b245e82966543dbe5